.gitpod.yml

```diff
@@ -0,0 +1,28 @@
+image:
+  file: .gitpod.dockerfile
+ports:
+- port: 3000 # Trace viewer example application
+- port: 9229 # Node.js debug port
+  onOpen: ignore
+- port: 9339 # Node.js debug port
+  onOpen: ignore
+- port: 6080 # VNC server
+  onOpen: ignore
+- port: 5900
+  onOpen: ignore
+- port: 8080 # Trace Compass server
+  onOpen: ignore
+tasks:
+- init: >
+    yarn download:server
+  command: >
+    yarn start:server
+- init: >
+    yarn &&
+    yarn download:sample-traces &&
+    yarn download:server
+  command: >
+    yarn start:browser ../ --hostname=0.0.0.0
```

Comment:
You should start-all here?

Reply:
We could but it would break something here. Each `- init` executes independently on Gitpod, and the log ends-up in a different terminal. This way we can more easily spot it, if e.g. something is wrong with the server, since the failed startup will be in its own terminal. 

Outside of Gitpod though, I think it's still useful to have the option to start both TC server and Trace Viewer example app, at once.